### PR TITLE
Add riscv-rt to workspace

### PR DESCRIPTION
**Pull request by romancardenas**
This PR adds [`riscv-rt`](https://github.com/rust-embedded/riscv-rt) to this workspace. I'm still working on fine-tuning the CI, let me know your opinion and I'll push it to the PR 😄

### CODE REVIEW

**Comment by romancardenas** [comment]
@rust-embedded/riscv good to go! Take a look and let me know if I forgot something

File: riscv-rt/README.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+[![crates.io](https://img.shields.io/crates/d/riscv-rt.svg)](https://crates.io/crates/riscv-rt)
+[![crates.io](https://img.shields.io/crates/v/riscv-rt.svg)](https://crates.io/crates/riscv-rt)
+[![Build Status](https://travis-ci.org/rust-embedded/riscv-rt.svg?branch=master)](https://travis-ci.org/rust-embedded/riscv-rt)
```
Comment (jessebraham) [inline riscv-rt/README.md:3]: Is Travis CI actually used? Doesn't appear to be, I'm guessing this is just an artifact from the past.

File: riscv-rt/macros/Cargo.toml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    "The RISC-V Team <risc-v@teams.rust-embedded.org>",
+    "Jorge Aparicio <jorge@japaric.io>",
+]
+categories = ["embedded", "no-std"]
+description = "Attributes re-exported in `riscv-rt`"
+documentation = "https://docs.rs/riscv-rt"
+keywords = ["riscv", "runtime", "startup"]
+license = "MIT OR Apache-2.0"
+name = "riscv-rt-macros"
+repository = "https://github.com/rust-embedded/riscv-rt"
```
Comment (jessebraham) [inline riscv-rt/macros/Cargo.toml:12]:
```suggestion
repository = "https://github.com/rust-embedded/riscv"
```

File: riscv/README.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
+[![crates.io](https://img.shields.io/crates/d/riscv.svg)](https://crates.io/crates/riscv)
+[![crates.io](https://img.shields.io/crates/v/riscv.svg)](https://crates.io/crates/riscv)
+[![Build Status](https://travis-ci.org/rust-embedded/riscv.svg?branch=master)](https://travis-ci.org/rust-embedded/riscv)
```
Comment (jessebraham) [inline riscv/README.md:3]: See above.

**Review by jessebraham — CHANGES_REQUESTED** [review]
Few little nitpicks but otherwise LGTM!

**Comment by romancardenas** [comment]
Thanks for your review @jessebraham ! I addressed all your comments and adapted the general `README.md` file from [`cortex-m`](https://github.com/rust-embedded/cortex-m)

**Review by jessebraham — APPROVED** [review]
Thanks for making those changes, LGTM!

**Pull request merged**